.github/workflows/code_style.yml

```diff
@@ -38,3 +38,30 @@ jobs:
         with:
           name: code_style_diff
           path: code_style_diff.patch
+          
+  Java:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v1
+        with:
+          java-version: '11.0.8' # The JDK version to make available on the path.
+          java-package: jdk # (jre, jdk, or jdk+fx) - defaults to jdk
+          architecture: x64 # (x64 or x86) - defaults to x64
+
+      - name: Check code style
+        run: |
+          wget -nc https://github.com/google/google-java-format/releases/download/google-java-format-1.9/google-java-format-1.9-all-deps.jar
+          java -jar google-java-format-1.9-all-deps.jar --set-exit-if-changed -i $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java")
+
+      - name: Create code style diff
+        if: failure()
+        run: |
+          java -jar google-java-format-1.9-all-deps.jar -r $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java") -i $(find . -type f -name "*.java" | grep ".*/inference-engine/ie_bridges/java/.*java")
```

Comment:
Why we need to runs?

Reply:
To change files code style and make diff